docs/environments/find-archived-environment.md

```diff
@@ -0,0 +1,21 @@
+---
+draft: true
+title: Find an Archived Environment
+intro: Sometimes you may need to access the details of an archived environment or see a list of environments in a project that have been archived. You can access details of all archived environments in a project owned by you.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness upper-left corner, click the Devopness logo to see a list of existing projects
+1. Select a `Project`
+1. Find the `Environments` card
+1. Click `View` in the `Environments` card to see a list of existing `Environments`
+1. On the upper-right corner of the `Environment` resources list, click `ARCHIVED ENVIRONMENTS`
+1. Find the `Archived Environment` you want to inspect and click `View`
+1. The `Environment` resources list will be displayed containing all the environment's running resources
```

Comment:
`running resources`
We don't have this concept in Devopness.
Do you mean in a server is stopped (via Devopness or outside Devopness), therefore no longer `running`, that server will not be displayed in this list? 🤔

Reply:
You are correct, that was a misunderstanding of the intent design from my behalf; will apply the patch below, if there are any regards after the changes, I'm happy to continue the discussion in this thread or a new one.

```suggestion
1. The `Environment` resources list will be displayed containing all the environment's resources
```